app/components/post_draft/post_draft.tsx

```diff
@@ -124,7 +144,7 @@ function PostDraft({
             >
                 {draftHandler}
             </KeyboardTrackingView>
-            <View nativeID={accessoriesContainerID}>
```

Comment:
I'm afraid that if you remove this, the native component will have problems with some calculations

Reply:
The nativeID is so the view is part of the keyboard accessory. Autocomplete shouldn't be part of the keyboard accessory, right?